modules/opensimplex/noise.cpp

```diff
@@ -0,0 +1,226 @@
+#include "noise.h"
+#include "core_string_names.h"
```

Comment:
This should be `core/core_string_names.h`, I removed `core/` from the include path. Please also add a line between `noise.h` (or `simplex_noise.h`) and other includes.